shim/README.md

```diff
@@ -0,0 +1,19 @@
+# cc-shim
```

Comment:
See my comments on the proxy README. The same comments and changes are needed on this file.